xonsh/base_shell.py

```diff
@@ -157,13 +157,6 @@ def readline(self, size=-1):
         self.std.seek(loc + len(s))
         return s
 
-    def write(self, s):
-        """Write a string to both streams and return the length written to the
-        in-memory stream.
-        """
-        self.std.write(s)
-        return self.mem.write(s)
```

Comment:
Hey @scopatz -- I'm happy to restore it, but there's also a method `self.write` defined here: https://github.com/xonsh/xonsh/pull/1810/files#diff-8534328692230305271758ac1f48a813R114 which doesn't return the `self.mem.write(s)` -- should the `write` on line 114 be removed instead?  I'll confess that I couldn't grok how the difference between them would propagate down the pipeline